src/components/attendee/VideoComms/scss/VideoCommsParticipant.module.scss

```
@@ -0,0 +1,55 @@
+@import "scss/constants";
```

Something more robust than `@import` and more structured than a flat constants file?

```suggestion
@use "scss/attendee/comms";
@use "scss/attendee/borders";
```